include/sway/ipc-json.h

```diff
@@ -4,13 +4,27 @@
 #include "sway/tree/container.h"
 #include "sway/input/input-manager.h"
 
+typedef void ipc_json_descriptor(struct sway_node *, json_object *);
+typedef ipc_json_descriptor *ipc_json_descriptors[NUM_NODE_TYPES];
```

Comment:
The one-char difference between `ipc_json_descriptor` and `ipc_json_descriptors` is confusing. Can we add a `_map` suffix or something?